src/web_or_worker_shell_read.js

```diff
@@ -45,11 +45,11 @@
     };
   }
 
-  readAsync = function readAsync(url, onload, onerror) {
+  readAsync = function(url, onload, onerror) {
     var xhr = new XMLHttpRequest();
     xhr.open('GET', url, true);
     xhr.responseType = 'arraybuffer';
-    xhr.onload = function xhr_onload() {
```

Comment:
why remove these names? (if it's for code size, I think they shouldn't matter, the minifier would remove them in a prod build)

Reply:
yeah it was an experiment for code size. I'm not sure how all these different things play together, but the minifier used by mono-wasm to generate ``dotnet.js`` didn't remove it.

I can re-add it, but all other functions also don't have names, so at least now it is consistent.